content/articles/ldnpydojos07e02.md

```diff
@@ -0,0 +1,175 @@
+Title: The London Python Dojo S07E02
+Date: 2015-10-03
+Short_summary: The We Are Wizards team was at the last London Python Code Dojo.
+Category: Events
+Authors: Gautier
+
+
+The We Are Wizards team was present at the last London Python Code Dojo held at
+Bank of America in Canary Wharf on the first of October.
+I'll report on the event in this post.
+<!-- PELICAN_END_SUMMARY -->
+
+<img width="100%" src="../images/dojocrowd.jpg">
+
+## What is a Code Dojo?
+
+You can find some information on the [London Python Code Dojo website](http://ldnpydojo.org.uk/).
+
+A Code Dojo is a "safe place to deliberately practise and develop your
+coding skills, and perhaps learn something new."
+
+It originally comes from Paris and is based on the following ideas:
+
+<blockquote><h4>From the <a href="http://codingdojo.org/cgi-bin/index.pl?WhatIsCodingDojo">Paris Code Dojo website</a></h4>
+
+<h4>Premise</h4>
+
+<ul>
+<li>acquiring coding skills should be a continuous process</li>
+</ul>
+
+<h4>Characteristics</h4>
+
+<ul>
+<li>non-competitive, collaborative, fun environment</li>
+<li>all skill levels are welcome</li>
+<li>safe to try new ideas</li>
+</ul>
+
+<h4>Types of meeting</h4>
+
+<h4>Prepared kata</h4>
+
+<p>
+A presenter shows how to solve the challenge from scratch, using TDD and
+BabySteps.
+Each step must make sense to everyone present for.
+People should interrupt only if they don't understand what is going on.
+</p>
+
+<h4>Randori kata</h4>
+
+<p>
+The challenge is solved by the coding pair (driver and copilot).
+Everyone present is invited to help.
+Each pair has a small (5 or 7 minutes) timebox to advance, using TDD and
+BabySteps.
+At the end of the timebox, the driver goes back to the audience, the copilot
+becomes driver and one of the audience step up to be copilot.
+</p>
+
+</blockquote>
+
+
+### The London style
+
+In London we start from the same premise and characteristics but we have a more
```

Comment:
"with the same premise, sticking to the characteriscs" - premise and characteristics don't have the same "type" so can't be used like this :)